### PR TITLE
add chinese readme

### PR DESCRIPTION
**Pull request by hnwyllmm1**
### Task Description

We want a README in Chinese.

### Solution Description

Add a Chinese REAMD doc.

### CODE REVIEW

**Comment by caifeizhi** [comment]
cc @a1iive @whhe

File: README_CN.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+
+OceanBase 数据库根据 Mulan 公共许可证版本 2 获得许可。有关详细信息，请参阅 [LICENSE](LICENSE) 文件。
+
+# 社区
+
+有以下加入社区的方法：
+
+* [中文论坛](https://ask.oceanbase.com/)
+* [Slack Workspace](https://join.slack.com/t/oceanbase/shared_invite/zt-1e25oz3ol-lJ6YNqPHaKwY_mhhioyEuw)
+* 钉钉群: 33254054 ([QR code](images/dingtalk.svg))
```
Comment (a1iive) [inline README_CN.md:136]: QR code -> 二维码

File: README_CN.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+    
+    唯一一个刷新了 TPC-C 记录（7.07 亿 tmpC）和 TPC-H 记录（1526 万 QphH @30000GB）的分布式数据库。
+
+- **实时分析**
+    
+    基于“同一份数据，同一个引擎”，同时支持在线实时交易及实时分析两种场景，“一份数据”的多个副本可以存储成多种形态，用于不同工作负载，从根本上保持数据一致性。
+
+- **高可用**
+    
+    独创“三地五中心”容灾架构方案，建立金融行业无损容灾新标准。支持同城/异地容灾，可实现多地多活，满足金融行业 6 级容灾标准（RPO=0，RTO< 30s），数据零丢失。
```
Comment (a1iive) [inline README_CN.md:56]: RTO < 30s -> RTO < 8s

File: README_CN.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+    <a href="https://github.com/oceanbase/oceanbase/releases/latest">
+        <img alt="license" src="https://img.shields.io/badge/dynamic/json?color=blue&label=release&query=tag_name&url=https%3A%2F%2Fapi.github.com%2Frepos%2Foceanbase%2Foceanbase%2Freleases%2Flatest" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase">
+        <img alt="stars" src="https://img.shields.io/badge/dynamic/json?color=blue&label=stars&query=stargazers_count&url=https%3A%2F%2Fapi.github.com%2Frepos%2Foceanbase%2Foceanbase" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase">
+        <img alt="forks" src="https://img.shields.io/badge/dynamic/json?color=blue&label=forks&query=forks&url=https%3A%2F%2Fapi.github.com%2Frepos%2Foceanbase%2Foceanbase" />
+    </a>
+    <a href="https://en.oceanbase.com/docs/community-observer-en-10000000000829617">
```
Comment (whhe) [inline README_CN.md:19]: Better to use https://en.oceanbase.com/docs/oceanbase-database

File: README_CN.md [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+</p>
+<p align="center">
+    <a href="https://github.com/oceanbase/oceanbase/blob/master/LICENSE">
+        <img alt="license" src="https://img.shields.io/badge/license-MulanPubL--2.0-blue" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase/releases/latest">
+        <img alt="license" src="https://img.shields.io/badge/dynamic/json?color=blue&label=release&query=tag_name&url=https%3A%2F%2Fapi.github.com%2Frepos%2Foceanbase%2Foceanbase%2Freleases%2Flatest" />
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase">
+        <img alt="stars" src="https://img.shields.io/badge/dynamic/json?color=blue&label=stars&query=stargazers_count&url=https%3A%2F%2Fapi.github.com%2Frepos%2Foceanbase%2Foceanbase" />
```
Comment (whhe) [inline README_CN.md:14]: Looks like the `stars` url is broken, is there any change in the badge api? @a1iive

**Review by whhe — COMMENTED** [review]
LGTM overall.

**Review by whhe — APPROVED** [review]

**Review by a1iive — APPROVED** [review]
LGTM

File: README_CN.md (hunk 4 above)
Comment (a1iive) [reply README_CN.md:14]: Github's problem :(

File: README_CN.md [hunk 5]
```diff
@@ -0,0 +1,137 @@
...
+    </a>
+    <a href="https://github.com/oceanbase/oceanbase/commits/master">
+        <img alt="last commit" src="https://img.shields.io/github/last-commit/oceanbase/oceanbase/master" />
+    </a>
+    <a href="https://join.slack.com/t/oceanbase/shared_invite/zt-1e25oz3ol-lJ6YNqPHaKwY_mhhioyEuw">
+        <img alt="Join Slack" src="https://img.shields.io/badge/slack-Join%20Oceanbase-brightgreen?logo=slack" />
+    </a>
+</p>
+
+[English](README.md)
```
Comment (caifeizhi) [inline README_CN.md:36]: [English](README.md) | [中文](README_CN.md)

**Pull request merged**